### PR TITLE
CAMEL-17662: make the test-infra documentation more visible to the end users

### PR DESCRIPTION
**Pull request by orpiske**

### CODE REVIEW

File: docs/user-manual/modules/ROOT/pages/testing.adoc [hunk 1]
```diff
@@ -31,8 +31,10 @@ class for all your configuration and routing without.
 |xref:components:others:test-spring-junit5.adoc[camel-test-spring-junit5] | *JUnit 5*: Used for testing Camel with Spring / Spring Boot
 |xref:components:others:test-cdi.adoc[camel-test-cdi] | Used for testing Camel on xref:components:others:cdi.adoc[CDI]
 
-|xref:components:others:testcontainers-junit5.adoc[camel-testcontainers-junit5] | *JUnit 5*: Camel Testcontainers extends standard camel test support providing a way to create and interact with containerized applications.
-|xref:components:others:testcontainers-spring-junit5.adoc[camel-testcontainers-spring-junit5] | *JUnit 5*: Camel Testcontainers extends standard camel Spring test support providing a way to create and interact with containerized applications.
+|xref:components:others:test-infra.adoc[camel-test-infra] | *Camel Test Infra*: Camel Test Infra is a set of modules that abstract the provisioning and execution of test infrastructure in a transparent way.
```
Comment (davsclaus) [inline docs/user-manual/modules/ROOT/pages/testing.adoc:34]: Can we mention testcontainers in the description here as I think this is what users would look for

**Review by davsclaus — APPROVED** [review]

File: docs/user-manual/modules/ROOT/pages/testing.adoc (hunk 1 above)
Comment (orpiske) [reply docs/user-manual/modules/ROOT/pages/testing.adoc:34]: Absolutely. I'll reword it.

**Review by oscerd — APPROVED** [review]

**Comment by davsclaus** [comment]
the docs cannot be build, eg see the error, or run mvn install in the docs folder to see the xref error

**Comment by orpiske** [comment]
> the docs cannot be build, eg see the error, or run mvn install in the docs folder to see the xref error

```
2022-02-09T13:58:15.8600155Z [ERROR]   manual:ROOT:testing.adoc (/home/runner/work/camel/camel/docs/user-manual/modules/ROOT/pages/testing.adoc) at line 34: xref:components:others:test-infra.adoc[camel-test-infra]
```

It seems strange. It is complaining about being unable to find an xref, but the xref seems there.

**Comment by orpiske** [comment]
@zregvart @djencks Hi, do you have any tips about what could be causing this?

**Comment by orpiske** [comment]
Moving it to the user-manual seems to do the trick.

**Comment by orpiske** [comment]
Looks good now, merging.

**Pull request merged**